docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,568 @@
+    required.  </para>
+
+    <para>What you will need in order to deploy an HPCC Systems instance with
+    Terraform:</para>
+
+    <itemizedlist>
+      <listitem>
+        <para>A Linux, MacOS, or Windows OS computer system.</para>
+      </listitem>
+
+      <listitem>
+        <para>A browser. Such as Chrome, or Firefox.</para>
+      </listitem>
+
+      <listitem>
+        <para>Git and a github account that you can access and clone the
+        repository.</para>
+      </listitem>
+
+      <listitem>
+        <para>An Azure account with sufficient permissions, rights, credits,
+        and credentials. To obtain one, go to www.azure.com or talk to your
+        manager if you believe that your employer might have a corporate
+        account.</para>
+      </listitem>
+
+      <listitem>
+        <para>A code editor of your choice. There are a few editors integrated
+        with Azure such as code, vi the Visual Editor, Nano, or you can choose
+        to use any another.</para>
+      </listitem>
+    </itemizedlist>
+
+    <para>The easiest option which also insures you have all the tools
+    required is to use Azure is the command portal. Assuming you have an Azure
+    account with all the appropriate credentials you can just go to the Azure
+    command portal</para>
+
+    <programlisting>https://portal.azure.com/</programlisting>
+
+    <para>If this is the first time you have accessed the cloud shell, Azure
+    will prompt you that storage is required for the cloud shell to persist
+    account settings and files. Click through the prompts to create the
+    storage. You should be presented with a shell. At this point, the cloud
+    shell should already be logged into to your Azure account. The major
+    advantage to using the cloud shell is that It will also have other
+    prerequisites installed for you, namely, git, kubectl, helm, and even an
+    editor. Otherwise in addition to installing the Azure CLI you would also
+    need to install the other prerequisites.</para>
+
+    <sect2 id="Workin_withTerraform">
+      <title>Working with the Terraform Code</title>
+
+      <para>The HPCC Systems Terraform Repository,
+      <emphasis>terraform-azurerm-hpcc</emphasis> is a code repository where
+      the HPCC Systems Terraform modules are stored. The Terraform repository
+      contains three modules required to instantiate an HPCC Systems cluster.
+      These are the network, storage, and AKS (Azure Kubernetes Service)
+      modules. The network module must be deployed first, followed by the
+      storage module, which is an optional module for deploying a storage
+      account. Finally, the AKS module, which is at the root of the
+      repository, can then be deployed. These modules will automatically call
+      other dependent modules upon initialization. </para>
+
+      <sect3>
+        <title>Cloning the Terraform Repository</title>
+
+        <para>You will want to clone the Terraform Repository,
```

Comment:
Clone the Terraform Repository, terraform-azurerm-hpcc hosted on the HPCC Systems GitHub account.